xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-index/src/main/java/org/xwiki/extension/index/internal/security/ExtensionAnalysisResult.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.extension.index.internal.security;
+
+import java.util.List;
+
+import static java.util.Comparator.comparingDouble;
+
+/**
+ * Store the known security issues for a given extension.
+ *
+ * @version $Id$
+ * @since 15.5RC1
+ */
+public class ExtensionAnalysisResult
```

Comment:
ExtensionAnalysisResult is a bit too generic as a name, it's not the only analysis we do on extensions.

What about ExtensionSecurityAnalysisResult ?